custom/up_nrhm/reports/asha_functionality_checklist_report.py

```diff
@@ -56,9 +56,9 @@ def rows(self):
             [8, _('Acting as DOTS provider')],
             [9, _('Holding or attending village/VHSNC meeting')],
             [10, _('Successful referral of the IUD, female sterilization or male '
-                 'sterilization cases and/or providing OCPs/Condoms')],
+                   'sterilization cases and/or providing OCPs/Condoms')],
             ['', _('Total of number of tasks on which ASHA reported being functional')],
-            ['', _('Total number of ASHAs who are functional on at least 60% of the tasks')],
+            ['', _('Total number of ASHAs who are functional on at least %s of the tasks' % '60%')],
```

Comment:
@czue 

If you mean to do ```60%%```, this solution will not work because this string is not formatted and it won't be escaped on the UI.